docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# Contribution Guidelines
+
+As all source{d} projects, this project follows the
+[source{d} Contributing Guidelines](https://github.com/src-d/guide/blob/master/engineering/documents/CONTRIBUTING.md).
+
+
+## Additional Contribution Guidelines
+
+In addition to the [source{d} Contributing Guidelines](https://github.com/src-d/guide/blob/master/engineering/documents/CONTRIBUTING.md),
+this project follows the guidelines described below.
+
+
+# Architecture
+
+The application is a go binnary that serves the statics and the API used for the UI.
+
+The statics (`html`, `js` and `css`) are generated by `yarn`, embedded in the go binary using bindata, and served from the binary itself.
+
+There are two main different ways to run the app:
+
+- using released artifacts, as [listed in the README.md#](../README.md#run-the-playground)
+- serving the app for development purposes, [building and running it from sources](#run-from-sources)
+
+
+# Configuration
+
+| Variable | Required | Default value | Meaning |
+| -- | -- | -- | -- |
+| `GITBASEPG_HOST` | | `0.0.0.0` | IP address to bind the HTTP server |
+| `GITBASEPG_PORT` | | `8080` | Port address to bind the HTTP server |
+| `GITBASEPG_SERVER_URL` | | `<GITBASEPG_HOST>:<GITBASEPG_PORT>` | URL used to access the application (i.e. public hostname) |
+| `GITBASEPG_DB_CONNECTION` | | `gitbase@tcp(localhost:3306)/none?maxAllowedPacket=4194304` | gitbase connection string |
+| `GITBASEPG_BBLFSHD_URL` | | `127.0.0.1:9432` | Address where bblfsh server is listening |
+| `GITBASEPG_ENV` | | `production` | Sets the log level. Use `dev` to enable debug log messages |
+
+
+# Development
+
+## Requirements
+
+You should already have [Go installed](https://golang.org/doc/install#install), and properly [configured the $GOPATH](https://github.com/golang/go/wiki/SettingGOPATH)
+
+```
+go version; # prints your go version
+go env GOPATH; # prints your $GOPATH path
+```
+
+The project must be under $GOPATH, as required by the Go tooling.
+You should be able to navigate into the source code by running:
+
+```
+cd $GOPATH/src/github.com/src-d/gitbase-playground
+```
+
+You need also [Yarn v1.x.x installed](https://yarnpkg.com/en/docs/install)
+
+```
+yarn --version; # prints your Yarn version
+```
+
+
+## Run from sources
+
+You need to satisfy all [project requirements](#requirements), and then run:
+
+```bash
+$ go get -d -u github.com/src-d/gitbase-playground/...
+$ cd $GOPATH/github.com/src-d/gitbase-playground
+$ make dependencies
+$ make serve
+```
+
+This will start a server locally, which you can access on [http://localhost:8080](http://localhost:8080)
+
+
+## Integration tests
+
+It will test that `gitbase` and `bblfshd` are running and available for the playground.
+
+It is needed to serve under `gitbase` a copy of the `https://github.com/src-d/gitbase-playground` repository
```

Comment:
"serve under `gitbase`" sound a bit confusing to me. This section would be much more clear if you copied the instructions from the quickstart doc.

Reply:
The quickstart has a different purpose; it's about dependencies, but this one is about the integration tests requirements.

Would you prefer @carlosms this other paragraph?
>_To let the tests succeed, it will be also needed to configure the running `gitbase` to serve a copy of the `https://github.com/src-d/gitbase-playground` repository._

Do you have any other suggestion ? :wink: